.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Build Distribution for Launcher
+        run: ./gradlew distForLauncher
+      - name: Unit Tests
+        run: ./gradlew unitTest
+      - name: API Documentation
+        run: ./gradlew javadoc
```

Comment:
would it make sense to extract those into dedicated jobs, too?

Reply:
hm, if we get the job dependencies with reuse going then yes, otherwise I would not blow up the job count as this job with multiple steps is still faster than the other two…